tests/test_api.rs

```diff
@@ -5699,10 +5699,9 @@ fn wasm_streaming_callback() {
 
   ws.finish();
   assert!(!scope.has_pending_background_tasks());
-  assert!(global.get(scope, name).unwrap().is_null());
-
-  scope.perform_microtask_checkpoint();
 
+  // We did not set wasm resolve callback so V8 uses the default one that
```

Comment:
Heads up for @andreubotella, this has changed in V8 in https://chromium-review.googlesource.com/c/v8/v8/+/3695584

I will add required bindings for that callback in a follow up